plugins/spi/SPIOutput.cpp

```diff
@@ -237,11 +235,11 @@ SPIOutput::~SPIOutput() {
 }
 
 
-string SPIOutput::GetDeviceLabel() const {
+std::string SPIOutput::GetDeviceLabel() const {
```

Comment:
Yeah sorry, let's put back the using, apologies for the confusion.

Reply:
no problem ;-) i change it back.

Reply:
Thanks